week2/exercise/Program.cs

```diff
@@ -0,0 +1,130 @@
+
+
+var builder = WebApplication.CreateBuilder(args);

```

Comment:
Seems like week2 has slipped into week3 somehow :)

Reply:
yes, i couldn't get rid of this. Could you please help me to remove this branch from here.